mmv1/third_party/terraform/tests/resource_dns_record_set_test.go

```diff
@@ -111,41 +102,13 @@ func TestAccDNSRecordSet_changeType(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				Config: testAccDnsRecordSet_basic(zoneName, "127.0.0.10", 300),
-				Check: resource.ComposeTestCheckFunc(
-					testAccCheckDnsRecordSetExists(
-						t, "google_dns_record_set.foobar", zoneName),
-				),
 			},
 			{
 				Config: testAccDnsRecordSet_bigChange(zoneName, 600),
-				Check: resource.ComposeTestCheckFunc(
-					testAccCheckDnsRecordSetExists(
-						t, "google_dns_record_set.foobar", zoneName),
-				),
-			},
-		},
-	})
-}
-
-func TestAccDNSRecordSet_ns(t *testing.T) {
```

Comment:
Why is this test case being removed? Can we keep it?

Reply:
The old resource was making `ResourceRecordSets.List` and verified the length to confirm the resource existence, as now we have direct resource method to Create/Get, i just replaced that test case with Create & Import for the same